scripts/editor-support.js

```diff
@@ -56,7 +56,9 @@ async function applyChanges(event) {
       attachEventListners(newMain);
       return true;
     }
-
+    if (element.matches('.fragment-wrapper')) {
```

Comment:
This is because I am removing the fragment block and keeping the wrapper. Because this is the case, the block can't be redecorated so on UE you wold only see a link instead of the content fragment unless you refresh.